Penteract/Assets/Scripts/EnemySpawnPoint.cpp

```diff
@@ -39,4 +45,11 @@ void EnemySpawnPoint::Update() {
 			}
 		}
 	}
+}
+
+void EnemySpawnPoint::KillEnemy() {
```

Comment:
This method should have another name, bc the spawn point is not killing the enemies, just incrementing the deadEnemies counter when each enemy dies, same with the WinLose class